wwwroot/inc/breed-hpprocurveN1178.php

```diff
@@ -213,7 +213,7 @@ function hpprocurveN1178Read8021QConfig ($input)
 				{
 				continue;
 				}
-			if (preg_match('/^([0-9]+)$/', trim($vlans), $matches))
+			if (preg_match('/^([Ttrk0-9]+)$/', trim($vlans), $matches))
```

Comment:
This would match the following:
* TTTTT
* krkrkr
* 123trtr456
* TtTtk

(and so on)
You likely mean something like `/^((?:[0-9]+)|(?:[Tt]rk[0-9]+))$/`